test/profiling/__init__.py

```diff
@@ -0,0 +1,19 @@
+class PerformanceEntry:
```

Comment:
Classes defined directly in `__init__.py`, consider placing into a separate file